### ISSUE TITLE
Java

### ISSUE THREAD

**Issue by MedoHamdani** [cdjalel/QuranKeyboard#3]
This issue appears when trying to enable the extension 

`(com.sun.star.uno.RuntimeException) { { Message = "[jni_uno bridge error] UNO calling Java method writeRegistryInfo: non-UNO exception occurred: java.lang.UnsupportedClassVersionError: dz/djalel/LO/comp/RegistrationHandler has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 52.0\X000ajava stack trace:\X000ajava.lang.UnsupportedClassVersionError: dz/djalel/LO/comp/RegistrationHandler has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 52.0\X000d\X000a\X0009at java.lang.ClassLoader.defineClass1(Native Method)\X000d\X000a\X0009at java.lang.ClassLoader.defineClass(Unknown Source)\X000d\X000a\X0009at java.security.SecureClassLoader.defineClass(Unknown Source)\X000d\X000a\X0009at java.net.URLClassLoader.defineClass(Unknown Source)\X000d\X000a\X0009at java.net.URLClassLoader.access$100(Unknown Source)\X000d\X000a\X0009at java.net.URLClassLoader$1.run(Unknown Source)\X000d\X000a\X0009at java.net.URLClassLoader$1.run(Unknown Source)\X000d\X000a\X0009at java.security.AccessController.doPrivileged(Native Method)\X000d\X000a\X0009at java.net.URLClassLoader.findClass(Unknown Source)\X000d\X000a\X0009at java.lang.ClassLoader.loadClass(Unknown Source)\X000d\X000a\X0009at java.lang.ClassLoader.loadClass(Unknown Source)\X000d\X000a\X0009at java.net.FactoryURLClassLoader.loadClass(Unknown Source)\X000d\X000a\X0009at java.lang.ClassLoader.loadClass(Unknown Source)\X000d\X000a\X0009at com.sun.star.comp.loader.RegistrationClassFinder.find(RegistrationClassFinder.java:53)\X000d\X000a\X0009at com.sun.star.comp.loader.JavaLoader.writeRegistryInfo(JavaLoader.java:376)\X000d\X000a at C:/cygwin64/home/buildslave/source/libo-core/bridges/source/jni_uno/jni_uno2java.cxx:780", Context = (com.sun.star.uno.XInterface) @0 } }`

**Comment by cdjalel** [comment]
Yes. The extension needs to be compiled with an older version of Java. Like
jdk 1.8.
Will do that soon incha Allah

**Comment by MedoHamdani** [comment]
Thank you.
By the way, your plug-in is similar to another one but for MS Word. You can find more details in Telegram, their channel or group called إدارة البيان البحثية
Also, wanted to contact you regarding an Arabic keyboard that doesn't have dots in it.
